manager/state/raft/raft.go

```diff
@@ -1142,15 +1145,30 @@ func (n *Node) send(ctx context.Context, messages []raftpb.Message) error {
 			continue
 		}
 
+		ch := make(chan struct{})
```

Comment:
Err, nevermind, this is what you explained in the description:

> Arguably this isn't the best way to do this, and it may be
cleaner to have a long-lived goroutine for sending to each node that's
supplied messages from a channel. I tried this and found it was quite
complicated to manage the lifecycles of these goroutines, so I prefer to
go with the solution that requires fewer changes for now.

I'm quite ignorant on this part of the code, but wouldn't it be easier to bucket messages by member then having `sendToMember` just take the list of messages and apply them in order?

Or encapsulate this in a struct (member?) so that each member has its own sending goroutine and a `Send()` function that just enqueues the message?